server/lib/nl/constants.py

```diff
@@ -192,6 +192,14 @@
             "Best": ["best",],
             "Worst": ["worst",],
         },
+        "Comparison": [
+            "compare(s|d)?",
+            "comparison",
+            "(is|has|have)( a| the)? \w+er",
+            # WARNING: These will conflate with Correlation
```

Comment:
do we return both comparison and correlation?

Reply:
Can you please validate that the Jan scripts work with this change?

Reply:
Done! Both scripts still work.